tests/Feature/Connect/SubmitLeaderboardEntryTest.php

```diff
@@ -60,9 +60,9 @@ public function testSubmitLeaderboardEntryEmptyLeaderboard(): void
         Carbon::setTestNow($now);
 
         /** @var Game $game */
-        $game = Game::factory()->create();
+        $game = $this->seedGame();
```

Comment:
This ensures the game is associated to a valid console ID, which prevents null exceptions trying to read `$game->system->id` during the session management/player metrics processing.